utils/_context/containers.py

```diff
@@ -63,6 +63,13 @@ def create_inject_volume():
     _get_client().volumes.create(_VOLUME_INJECTOR_NAME)
 
 
+def start_container(container, replay=False):
```

Comment:
Could you just change the prototype of `TestedContainer.start` by adding `replay` argument, and calling `post_start` at the end?

Reply:
You can even change `post_start` name to `_post_start` (python convention for private methods)

Reply:
I made the change 👍

Reply:
Looks like there are many places that call `.start` directly, so to avoid too many code changes I'll leave the `post_start` out of `start` and out of the threads as it was before.